content/influxdb/cloud-dedicated/reference/sql/functions/time-and-date.md

```diff
@@ -360,6 +361,276 @@ GROUP BY _time, room
 {{% /expand %}}
 {{< /expand-wrapper >}}
 
+## date_bin_wallclock
+
+Calculates time intervals using the timezone of a specified time value and
+returns the start of the interval nearest to the specified timestamp.
+Use `date_bin_wallclock` to downsample time series data by grouping rows into
+time-based "bins" or "windows" that are based off "wall clock" times in a
+specific timezone and applying an aggregate or selector function to each window.
+
+### Time zone shifts
+
+Many regions use time zone shifts (such as Daylight Saving Time (DST) and British
```

Comment:
"Daylight Saving Time" and "British Summer Time" aren't equivalents. DST refers to the general concept of adjusting the local time to make the most of the daylight hours. BST is a time zone offset identifier. I think this should be reworded a little to express that.

Reply:
@mhilton Thanks. Is there a generalized term you use in the UK to represent the 1 hour seasonal offset? Is it just Daylight Savings Time? In the US, we have the general "DST," but we also have timezone-specific offset identifiers (Pacific Standard Time (PST) vs Pacific Daylight Time (PDT)).

Reply:
Yes it's very similar, but in the UK we only have to worry about one time-zone so they can be used haphazardly. The concept is known as "Daylight Saving Time" which changes the timezone from "GMT" to "BST".